cmd/bblfsh/client.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/Sirupsen/logrus"
 	"google.golang.org/grpc"
 	"gopkg.in/bblfsh/sdk.v0/protocol"
-	"srcd.works/go-errors.v0"
+	"gopkg.in/src-d/go-errors.v0"
```

Comment:
Just wanted to comment that there is a `v1` too, linking to `v1.0.0`, I'm not sure if we must use `v0`.

Reply:
The reason for using `v0` is that we are already using it in other parts of the repo, but looking at the difference betweent the two, there are none, so I'll upgrade to `v1`.